docs/interpreter/python.md

```diff
@@ -47,13 +47,13 @@ You can leverage [Zeppelin Dynamic Form]({{BASE_PATH}}/manual/dynamicform.html)
 Example : 
```

Comment:
@hriviere Thanks for the quick response! 👍  It seems most of them I mentioned are addressed except adding a new line in here. Could you check this ?